### PR TITLE
Introducing AET changelog.

### PR DESCRIPTION
**Pull request by Skejven**
Here is a pull request with an idea to track significant changes through the AET history.
The idea is to update `CHANGELOG.md` file every time an important pull request is merged into master so that we can easily show what bugfixes/enchancments/new features are delivered within new version.
Contribution and feedback are welcome.
___

I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,6 +41,11 @@ This documentation after update is ported to [AET wiki](https://github.com/Cogni
 When updating documentation please update proper markdown pages in [documentation](https://github.com/Cognifide/aet/tree/master/documentation) module following [instructions](https://github.com/Cognifide/aet/blob/master/documentation/README.md) and include it with your pull request.
 After your pull request is merged, wiki pages will be updated.
 
+## Changelog
+When changing or fixing some important part of AET, please remember to update [Changelog](https://github.com/Cognifide/aet/blob/master/CHANGELOG.md).
+Your entry should be enlisted in `Unreleased` section. It will be moved to appropriate release notes when released.
+Please use convention `- [PR-ABC](https://github.com/Cognifide/aet/pull/ABC) - short description of the PR.`.
```
Comment (wiiitek) [inline CONTRIBUTING.md]: Maybe `[PR-123]` would be better for the convention example? What do you think?

File: pom.xml [hunk 2]
```diff
@@ -623,14 +623,13 @@
...
               <!-- Files that don't require a license header -->
-              <exclude>**/README.md</exclude>
               <exclude>**/LICENSE</exclude>
               <exclude>**/NOTICE</exclude>
-              <exclude>CONTRIBUTING.md</exclude>
               <exclude>.travis.yml</exclude>
 
               <exclude>**/target/**</exclude>
 
+              <exclude>**/*.md</exclude>
```
Comment (wiiitek) [inline pom.xml]: If we are to exclude all `.md` files from rat plugin then we could also remove the exclusions from **documentation** module pom.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# AET Releases
+All notable changes to AET will be documented in this file.
+
+## Unreleased
+**List of changes that are finished but not yet released in any final version.**
+- [PR-41](https://github.com/Cognifide/aet/pull/41) - [Click](https://github.com/Cognifide/aet/wiki/ClickModifier)
+ and [Hide](https://github.com/Cognifide/aet/wiki/HideModifier) modifiers allows to use css selectors as parameters. 
+ Added [Replace Text](https://github.com/Cognifide/aet/wiki/ReplaceTextModifier) modifier and it's tests.
+- [PR-39](https://github.com/Cognifide/aet/pull/39) - fixed filterint w3cHtml issues.
```
Comment (subiron) [inline CHANGELOG.md]: typo "filterint"

File: CONTRIBUTING.md (hunk 1 above)
Comment (Skejven) [inline CONTRIBUTING.md]: I'd rather not put numbers here. Is `ABC` not clear?

File: pom.xml (hunk 2 above)
Comment (Skejven) [inline pom.xml]: Good point, removed.

**Pull request merged**